### ISSUE TITLE
Ability to zoom

### ISSUE THREAD

**Issue by jjahanip** [Neural-Systems-at-UIO/VisuAlign#203]
We are working with high-res images which requires zoomed in part of the brain to find the exact point to align the atlas.
I was wondering if you could add the feature to be able to zoom into the image.

Thanks.

**Comment by Tevemadar** [comment]
Yes, this is an understandable request, we have encountered it in the past already. There are technical and practical issues though: the simple built-in image handling what VisuAlign uses from the JavaFX UI framework only pretends it can deal with very large images, actually it resizes them. This is completely invisible now, but it would become a problem when zooming in.
While of course the API could be replaced with another one, that's a significant rewrite as it wouldn't be a built-in anymore, and thus it would need interfacing with the framework, plus of course implementation of pan-and-zoom controls. We simply don't have the resources to do this in a short timeframe.

Side note, which may be relevant: there is an online counterpart of VisuAlign, which is under active development, and it's available for use in the [EBRAINS Collaboratory](https://wiki.ebrains.eu/bin/view/Collabs/) environment. That variant of the tool relies on tiled multiresolution images (DeepZoom format in particular, a converter service is part of the [workflow](https://wiki.ebrains.eu/bin/view/Collabs/quint-demo)), and has no particular limitation on image resolution. Here is an [example](https://localizoom.apps.hbp.eu/filmstripzoom.html?atlas=WHS_SD_Rat_v4_39um&series=https://data-proxy.ebrains.eu/api/v1/public/buckets/img-c25b1258-d6ca-4528-813d-5f33dff10562/12877_PtP_BDA_IR-WHSSDv4-nonlin-fix.json&pyramids=buckets/img-c25b1258-d6ca-4528-813d-5f33dff10562&tools&nl).

It's just the first link from https://search.kg.ebrains.eu/instances/cfd55c91-bc2c-4f15-a04e-17d50ed232e7 with the markers enabled. (Having such dataset page and sharing data is not a requirement for using the online tools, this paragraph is just an advertisement)

**Comment by Majpuc** [comment]
Hi @jjahanip 
Does this answer your question?
